lelkek.FrogBox/store

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html charset="utf8">
+<html>
+<head>
+<title>FrogBox Store</title>
+<link rel="stylesheet" href="./css/web.css">
+<link rel="apple-touch-icon" sizes="57x57" href="./apple-icon-57x57.png">
+<link rel="apple-touch-icon" sizes="60x60" href="./apple-icon-60x60.png">
+<link rel="apple-touch-icon" sizes="72x72" href="./apple-icon-72x72.png">
+<link rel="apple-touch-icon" sizes="76x76" href="./apple-icon-76x76.png">
+<link rel="apple-touch-icon" sizes="114x114" href="./apple-icon-114x114.png">
+<link rel="apple-touch-icon" sizes="120x120" href="./apple-icon-120x120.png">
+<link rel="apple-touch-icon" sizes="144x144" href="./apple-icon-144x144.png">
+<link rel="apple-touch-icon" sizes="152x152" href="./apple-icon-152x152.png">
+<link rel="apple-touch-icon" sizes="180x180" href="./apple-icon-180x180.png">
+<link rel="manifest" href="/manifest.json">
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,700" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Montserrat:700" rel="stylesheet">
+<meta name="msapplication-TileColor" content="#ffffff">
+<meta name="msapplication-TileImage" content="/ms-icon-144x144.png">
+<meta name="theme-color" content="#ffffff">
+<meta name="author" content="auw46">
+<meta name="Description" content="FrogBox Entertainment is a small independent company based out of Portland, Maine.">
+<meta name="Keywords" content="frog, box, frogbox, entertainment, gammasquared, scyfa, arena, arena of, the elders, the, elders, butthole">
+
+<!--
+_____/\\\\\\\\\\\______________/\\\________/\\\__________________/\\\\\\\\\__________________/\\\\\\\\\_        
+ ___/\\\/////////\\\___________\/\\\_______\/\\\_______________/\\\////////________________/\\\////////__       
+  __\//\\\______\///____________\/\\\_______\/\\\_____________/\\\/_______________________/\\\/___________      
+   ___\////\\\___________________\/\\\_______\/\\\____________/\\\________________________/\\\_____________     
```

Comment:
After learning the meaning of succ thanks to Eddie (I didn't bother googling it because I thought it was an inside joke), why the fuck do you have that written in your html files.